### ISSUE TITLE
[11045] Project Module

### ISSUE THREAD

**Issue by Aycorn** [DeskDirector/UserVoice#123]
Similar to the current modules on the left hand pane, including a Project Module:

![image](https://user-images.githubusercontent.com/30588784/43750093-e1f9e010-9a4a-11e8-9585-9568b65a9806.png)

This would provide easy access to information such as:

- Breakdown of phases
- Budgeted hour breakdowns
- Project Ticket Issues

A good indication of what information would be helpful is primarily listed under the "Work Plan" tab of any project within ConnectWise.

**Comment by AirLHull** [comment]
Morning Guys, what are your opinions on this? We think (Especially when we on-boarding a new client) being able to show the Phases being worked on and closed and the specific Project points shows great innovation and gives the new client a great understanding of what DD can do for them!

Not to mention current clients that have a normal project (e.g. New Phone System), client will most likely want to see how far the Project team have got with the project, and will save multiple e-mails / calls etc.

Only primary contacts will normally need to see the projects however i was thinking more of another Permission tab for the user called "Projects", and similar to tickets "My Projects" "All Projects" and then "Closed" "Open", same as the all Tickets feature.

We are keen to get this approved, please do let me know if a discussion is needed.

**Comment by Nness** [comment]
As I have stated in other issues, we have quarterly goal. The information we gathered here is to help determine what we will be focus on in next quarter.

For CW projects, different MSP have different feeling on that. Even when they use it, whether they think it is proper tools or not it is still questionable. 

We would like to seek an middle ground where it is applicable for both Autotask and ConnectWise.

**Comment by Aycorn** [comment]
Similar Request: Service Ticket #14223 - Project Board Access for simplified Project Management progress report

_It would be great to give an individual at a client access to the project board to see a simplified chart (think gantt and/or like a delivery status like UPS/FedEx) with where there project is and allow to provide and receive updated from the client._

**Comment by AirLHull** [comment]
Any other thoughts on this?

**Comment by Aycorn** [comment]
Similar Request: Service Ticket #20540 - DeskDirector Access to Projects

**Comment by hulsville** [comment]
ConnectWise now has project visibility in their own user portal. This is something more and more customers are asking for and is needed in DeskDirector. We will be testing out ConnectWise's new customer portal for this reason alone.

**Comment by Lowell-Syncrety** [comment]
Yes, I am looking for a tool to properly update customers with project status. Right now it's a manual email we have to send out copying information from CW Manage. I just looked at the new CWM user portal and the Projects section is basic, but pretty effective for customers. It's very simple and easy for them to review.

We'd love to see something like this in DeskDirector as well. @Aycorn thanks for the suggestion.

**Comment by harismic** [comment]
I recently opened a ticket with support regarding this feature. This would be hugely beneficial for us and our clients to have a PM module to track and update projects on that Arent commingled.
